servers/partner-ci/builders/rescan_ds.py

```diff
@@ -0,0 +1,410 @@
+#!/usr/bin/env python
+"""Copyright 2016 Mirantis, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License"); you may
+not use this file except in compliance with the License. You may obtain
+copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+License for the specific language governing permissions and limitations
+under the License.
+"""
+
+import argparse
+import atexit
+import logging as log
+import ssl
+import sys
+
+from os import environ
+
+import paramiko
+
+from pyVim import connect
+
+from pyVmomi import vim
+from pyVmomi import vmodl
+
+import requests
+
+requests.packages.urllib3.disable_warnings()
+log.getLogger("requests").setLevel(log.WARNING)
+log.basicConfig(format='%(message)s', level=log.INFO)  # %(levelname)s:
+
+
+class NotFoundException(Exception):
+    """Raise when some object cannot be found."""
+
+    pass
+
+
+class Victl(object):
+    """VMware base actions."""
+
+    _service_instance = None
+    content = None
+
+    def __init__(self, host, user, password, port):
+        """Create ssl context."""
+        try:
+            # workaround https://github.com/vmware/pyvmomi/issues/235
+            context = ssl.SSLContext(ssl.PROTOCOL_TLSv1)
+            context.verify_mode = ssl.CERT_NONE
+            self._service_instance = connect.SmartConnect(host=host,
+                                                          user=user,
+                                                          pwd=password,
+                                                          port=int(port),
+                                                          sslContext=context)
+
+            if not self._service_instance:
+                raise Exception('Could not connect to the specified host using'
+                                ' specified username and password')
+
+            atexit.register(connect.Disconnect, self._service_instance)
+
+            self.content = self._service_instance.RetrieveContent()
+
+        except vmodl.MethodFault as e:
+            raise Exception('Caught vmodl fault: ' + e.msg)
+
+    def get_dc_object(self, datacenter):
+        """Return datacenter object with specified name."""
+        for dc in self.content.rootFolder.childEntity:
+            if dc.name == datacenter:
+                return dc
+
+        raise NotFoundException("Can not find dc "
+                                "'{dc_name}'".format(dc_name=datacenter))
```

Comment:
we have a specific check for jjb configuration to avoid unresolved jjb's variables, like {var1}. So we need to change template for python's .format() to {} and {0}, {1}, etc.